tools/grab_micro_posts.py

```
@@ -25,12 +31,15 @@
     return h.handle(html_content)
 
 def download_image(url, output_path):
-    response = requests.get(url)
-    if response.status_code == 200:
+    try:
+        response = requests.get(url)
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)